README.md

```diff
@@ -5,182 +5,212 @@
 
 <div align=center>
 
-<h3>pgai allows you to develop RAG, semantic search, and other AI applications directly in PostgreSQL</h3>
+<h3>pgai is your PostgreSQL-powered data layer for AI applications</h3>
```

Comment:
This top line matters a lot. 

We could go a few ways:
- Build AI applications with PostgreSQL.
- Or describe pgai at a high level, like we do above.

Reply:
```suggestion
<h3> Power your AI applications with PostgreSQL
 </h3>
```

Reply:
data layer is too vague imo I like the previous header but avthars suggestion works too.